sig-scheduling/annual-report-2024.md

```diff
@@ -12,36 +12,39 @@
    - Governance and leadership changes
 -->
 
+- **The performance improvement**
+  - Have put a lot of effort into QueueingHint ([KEP-4247](https://github.com/kubernetes/enhancements/tree/master/keps/sig-scheduling/4247-queueinghint)), which enhances the scheduler's retrying efficiency.
+  - Introduced the asynchronous preemption ([KEP-4832](https://github.com/kubernetes/enhancements/tree/master/keps/sig-scheduling/4832-async-preemption)), which improves the scheduling throughput with the preemption.
+  - Improved [the internal scheduling performance test (scheduler-perf)](https://github.com/kubernetes/kubernetes/tree/master/test/integration/scheduler_perf) to cover more scenarios and alert us when the degradation is introduced.
+- **DRA**: Improved the DRA scheduling with the structured parameters ([KEP-4381](https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/4381-dra-structured-parameters#kube-scheduler)).
+
 2. Are there any areas and/or subprojects that your group needs help with (e.g. fewer than 2 active OWNERS)?
 
+None.
 
 3. Did you have community-wide updates in 2024 (e.g. KubeCon talks)?
 
 <!--
   Examples include links to email, slides, or recordings.
 -->
 
-4. KEP work in 2024 (v1.30, v1.31, v1.32):
-<!--
-   TODO: Uncomment the following auto-generated list of KEPs, once reviewed & updated for correction.
+- [Kubecon EU 2024: SIG_Scheduling Intro & Deep Dive](https://sched.co/1YhjR)
+- [Kubecon NA 2024: SIG_Scheduling Intro & Updates](https://sched.co/1hovV)
 
-   Note: This list is generated from the KEP metadata in kubernetes/enhancements repository.
-      If you find any discrepancy in the generated list here, please check the KEP metadata.
-      Please raise an issue in kubernetes/community, if the KEP metadata is correct but the generated list is incorrect.
--->
+4. KEP work in 2024 (v1.30, v1.31, v1.32):
 
-<!-- 
   - Alpha
     - [4816 - DRA Prioritized List](https://github.com/kubernetes/enhancements/tree/master/keps/sig-scheduling/4816-dra-prioritized-list) - v1.32
     - [4832 - Asynchronous Preemption](https://github.com/kubernetes/enhancements/tree/master/keps/sig-scheduling/4832-async-preemption) - v1.32
 
   - Beta
     - [3633 - Introduce MatchLabelKeys and MismatchLabelKeys to PodAffinity and PodAntiAffinity](https://github.com/kubernetes/enhancements/tree/master/keps/sig-scheduling/3633-matchlabelkeys-to-podaffinity) - v1.31
+    - [4247 - Per-plugin callback functions for efficient requeueing in the scheduling queue](https://github.com/kubernetes/enhancements/blob/master/keps/sig-scheduling/4247-queueinghint/README.md) - v1.32
```

Comment:
It was lacking. The metadata is fixed at https://github.com/kubernetes/enhancements/pull/5050.